src/ddo/DDO.ts

```diff
@@ -188,7 +192,13 @@ export class DDO {
                     ...metadata.main
                 } as any
             }
-        } as Service)
+        } as Service
+        if (nftAttributes) {
+            metadataService.attributes.main['ercType'] = nftAttributes.ercType
+            metadataService.attributes.main['nftType'] = nftAttributes.nftType
```

Comment:
is there a need to add this to the metadata service since they already exist in the `nft-access`, `nft-sales` services?

Reply:
This duplication of the information on the DDO usually leads to problems down the line.

Reply:
@aaitor 

Probably best to discuss this in the spec and fix there first. There's a ton of duplication with conditions and stuff.
I guess these attributes should be removed from services and kept here, I don't think it currently makes sense to have different nft type for sales and access...